schemas/v3.schema.graphql

```diff
@@ -266,6 +267,8 @@ type FlashLoan @entity {
   target: Bytes!
   amount: BigInt!
   totalFee: BigInt!
+  lpFee: BigInt!
```

Comment:
split up the fee, called it fee instead of `premium` to be consistent with the totalFee we already had